infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -126,28 +74,41 @@ public void delete(@Nonnull ConflictTxStream key, Long globalAddress, @Nonnull R
      * @return global address
      */
     public Long getIfPresent(ConflictTxStream conflictKey) {
-        return conflictCache.getIfPresent(conflictKey);
+        return cacheHashtable.get(conflictKey);
+    }
+
+    private long firstAddress() {
+        if (cacheEntries.peek() == null)
+            return Address.NOT_FOUND;;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/manual/annymartinez.m/CorfuDB/pullRequest?prid=4748148)